lib/index.ts

```diff
@@ -18,6 +18,7 @@ import * as Utils from './utils';
 export * from './build-task';
 export * from './errors';
 export * from './local-image';
+export { PathUtils };
```

Comment:
How is this made importable? `import { PathUtils } from 'resin-multibuild`?

Reply:
Yeah exactly, or 
```
import * as multibuild from 'resin-multibuild';

multibuild.PathUtils.contains(...)
```